### PR TITLE
1176: Configure locales in build config

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Add the available locales to the build config and use them to configure the app locales.

### Proposed changes

- Add `appLocales` to build config
- Fix builder.dart script
- Use `appLocales` to configure the app locales

### Side effects

None.

### Resolved issues

Fixes: #1176.

### CODE REVIEW

**Comment by steffenkleinle** [comment]
To test this, you can set the system language to English and change one translation in `app_en.arb` to something else. For bayern this should not show up in the app while for nürnberg it should.

File: frontend/l10n.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 arb-dir: lib/l10n
-template-arb-file: app_en.arb
+template-arb-file: app_de.arb
```
Comment (f1sh1918) [inline frontend/l10n.yaml:2]: why this change?

Comment (steffenkleinle) [reply frontend/l10n.yaml:2]: Basically it doesn't really matter as long as the translation files are identical/have the same keys. But since (at least atm) all our apps/white labels have german as a language but not every app has english I figured using german as a default scheme makes more sense.
